packages/storybook/package.json

```diff
@@ -4,33 +4,47 @@
   "private": true,
   "scripts": {
     "dev": "storybook dev -p 6006",
-    "build": "storybook build --docs",
+    "build": "storybook build",
     "preview-storybook": "serve storybook-static",
     "clean": "rm -rf .turbo && rm -rf node_modules",
     "lint": "eslint ./stories/*.stories.tsx"
   },
   "prettier": "@dailydotdev/prettier-config",
   "dependencies": {
     "@dailydotdev/shared": "*",
+    "@tanstack/react-query": "^4.36.1",
```

Comment:
Wait do you need this here for sb?

Reply:
Yes, extension component are consuming a lot of business logic inside and `QueryClientProvider` is mandatory in order to show the component :(